_example/lambda.go

```diff
@@ -4,20 +4,64 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"os"
 	"strconv"
 
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go/aws"
 	lumigotracer "github.com/lumigo-io/go-tracer-beta"
+	"golang.org/x/net/context/ctxhttp"
 )
 
 type MyEvent struct {
 	Name string `json:"name"`
 }
 
 func HandleRequest(ctx context.Context, name MyEvent) (events.APIGatewayProxyResponse, error) {
+	client := &http.Client{
+		Transport: lumigotracer.NewTransport(http.DefaultTransport),
+	}
+	cfg, _ := config.LoadDefaultConfig(context.Background(), config.WithHTTPClient(client))
+	cfg.Region = "us-east-1"
+	// testing S3
+	svc := s3.NewFromConfig(cfg)
+	_, err := svc.ListBuckets(context.Background(), &s3.ListBucketsInput{})
+	if err != nil {
+		return events.APIGatewayProxyResponse{Body: "", StatusCode: 500}, err
+	}
+
+	// testing SSM
+	ssmClient := ssm.NewFromConfig(cfg)
+	input := &ssm.GetParameterInput{
+		Name: aws.String("parameter-name"),
+	}
+	_, err = ssmClient.GetParameter(context.Background(), input)
+	if err != nil {
+		return events.APIGatewayProxyResponse{Body: "ssm error", StatusCode: 500}, err
+	}
+
+	// testing ctxhttp
+	req, err := http.NewRequest("GET", "https://www.google.com", nil)
```

Comment:
just to make sure, because maybe I missed something in the code - can you validate that the body of the response is valid?